frontend/scss/components/molecules/table-component.scss

```diff
@@ -0,0 +1,180 @@
+/*
+
+##################################
+### ORGANISM: table component ###
+##################################
+
+### INFO:
+
+*/
+
+@import '../../extends';
```

Comment:
Use absolute paths instead in all files. So this would get to be: `@import '_extends.scss';`

Reply:
Done